### DISCUSSION TITLE
Playback is not working on Kodi 18.7 Leia on RetroPie

### DISCUSSION THREAD

**Discussion by jgoodwi32** [CastagnaIT/plugin.video.netflix#1299] [Q&A]
I am running into the MSL error whenever I try to play anything on Netflix. In an attempt to solve the issue, I updated widevine to its latest version; logged out and logged back in, and reinstalled the add-on. I am not sure what else I can do here. I don't have the option to change the MSL Manifest Version on 18.  I will attach the section of the log that pertains to this issue. Netflix was working just fine a few days ago, so I'm not sure what may have happened here. 

https://pastebin.com/GnL5fdve

**Comment by DerGanjalf** [comment]
Same problem on LibreElec, Kodi 18.9, Raspberry Pi 3. Playback was working fine 2 days ago, now I receive the error "This title is not available to watch instantly". No client changes occured in this period (e.g. Widevine Update). I got an other Pi, which is configured the same way and has exactly the same issue. Maybe a change on Netflix' end?
Debug Log: http://ix.io/3OqN

**Reply by jgoodwi32** [reply]
A temporary workaround was posted on the forum, starting at comment #2907:

https://forum.kodi.tv/showthread.php?tid=329767&page=194
